_test/test_TF_components/test_tobyfit_cuts.m

```diff
@@ -66,7 +66,7 @@
             obj.fe_arr = fe_arr;  % three short const-E cuts, 140-160, 160-180, 180-200 meV
             obj.rb_arr = rb_arr;    % two const-Q cuts, 2-10 meV
 
-            tol_sig = 0.25;        % tolerance as multiple of st. dev. of reference value
+            tol_sig = 0.5;        % tolerance as multiple of st. dev. of reference value
```

Comment:
I think not taken in isolation this changes much. It's of note here because it is changing, but going forward I'm not sure the reasoning is necessarily relevant.